partner_address_format_domestic/tests/test_partner_address_format_domestic.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestPartnerAddressFormatDomestic(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.country_us = cls.env["res.country"].search([("code", "=", "US")], limit=1)
+        cls.country_jp = cls.env["res.country"].search([("code", "=", "JP")], limit=1)
+        cls.country_us.address_format_domestic = (
+            "%(street)s\n%(street2)s\n%(city)s\n%(state_code)s\n%(zip)s"
+        )
+        cls.country_jp.address_format_domestic = (
+            "%(street)s\n%(street2)s\n%(city)s\n%(state_code)s\n%(zip)s"
+        )
+        cls.new_company = cls.env["res.company"].create(
+            {
+                "name": "Test Company",
+                "country_id": cls.country_us.id,
+            }
+        )
+        cls.partner_us = cls.env["res.partner"].create(
+            {
+                "name": "John Doe",
+                "country_id": cls.country_us.id,
+            }
+        )
+
+        cls.partner_jp = cls.env["res.partner"].create(
+            {
+                "name": "Yamada Taro",
+                "country_id": cls.country_jp.id,
+            }
+        )
+
+    def test_us_partner_uses_domestic_format(self):
+        self.env.company = self.new_company
+        format_us = self.partner_us._get_address_format()
+        self.assertEqual(
+            format_us,
+            self.country_us.address_format_domestic,
```

Comment:
Try to use static values for tests when possible.
```suggestion
            "%(state_code)s\n%(zip)s",
```